src/oas/transformers/schema/keywords/format.ts

```diff
@@ -15,6 +15,7 @@ const ranges = {
 
 const createNumberFormatter = (min: number, max: number): Converter => {
   return schema => {
+    schema['x-stoplight'] = { explicitProperties: Object.getOwnPropertyNames(schema) };
```

Comment:
```suggestion
    schema['x-stoplight'] = { explicitProperties: Object.keys(schema) };
```
we don't care about enumerable keys, they don't exist in JSON so anything non-enumerable would be set by us and we prob don't want to include that in the output.

Reply:
good call out - I updated to use keys